static/app/views/profiling/profileSummary/index.tsx

```diff
@@ -307,6 +309,7 @@ function ProfileSummaryPage(props: ProfileSummaryPageProps) {
             </ProfileVisualization>
             <ProfileDigest>
               <div>TODO: Profile Digest</div>
+              <SlowestProfileFunctions transaction={transaction ?? ''} />
```

Comment:
Should we send an error somewhere? `transaction` should always be defined on this page.

Reply:
Yeah, let me log an exception here.